From 6ce2a58e2acdde4000669cda0888fea6a75b9ccd Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 21 Sep 2018 00:52:58 +0200
Subject: [PATCH 16/17] lineage: Use add_json_str_omitempty directive for
 TARGET_LD_SHIM_LIBS

* The code is ifdef'ed out, so let's match that expecation here

Change-Id: I59c8e276c3dd28c4f46222f66d626e48b2aa6551

# Conflicts:
#	build/soong/soong_config.mk
---
 build/soong/soong_config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/soong/soong_config.mk b/build/soong/soong_config.mk
index 9dbdece0..f9b19977 100644
--- a/build/soong/soong_config.mk
+++ b/build/soong/soong_config.mk
@@ -13,7 +13,7 @@ $(call add_json_str, Java_Source_Overlays, $(JAVA_SOURCE_OVERLAYS))
 $(call add_json_str, Specific_camera_parameter_library, $(TARGET_SPECIFIC_CAMERA_PARAMETER_LIBRARY))
 $(call add_json_str_omitempty, Target_process_sdk_version_override, $(TARGET_PROCESS_SDK_VERSION_OVERRIDE))
 $(call add_json_str_omitempty, Target_specific_headers_include_dir, $(TARGET_SPECIFIC_HEADER_PATH))
-$(call add_json_str, Target_shim_libs, $(TARGET_LD_SHIM_LIBS))
+$(call add_json_str_omitempty, Target_shim_libs, $(TARGET_LD_SHIM_LIBS))
 $(call add_json_bool, Uses_generic_camera_parameter_library, $(if $(TARGET_SPECIFIC_CAMERA_PARAMETER_LIBRARY),,true))
 $(call add_json_bool, Uses_nvidia_enhancements, $(filter TRUE,$(NV_ANDROID_FRAMEWORK_ENHANCEMENTS)))
 $(call add_json_bool, Uses_qcom_bsp_legacy, $(filter true,$(TARGET_USES_QCOM_BSP_LEGACY)))
-- 
2.17.1

